onebusaway-api-webapp/src/test/java/org/onebusaway/api/actions/api/where/ScheduleForRouteActionTest.java

```diff
@@ -0,0 +1,352 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.onebusaway.api.actions.api.where;
+
+
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Matchers.anyListOf;
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.anyMapOf;
+import static org.onebusaway.transit_data_federation.testing.UnitTestingSupport.*;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintWriter;
+import java.math.BigDecimal;
+import java.util.*;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.struts2.rest.DefaultHttpHeaders;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.onebusaway.api.actions.siri.StopPointsV2Action;
+import org.onebusaway.api.actions.siri.impl.RealtimeServiceV2Impl;
+import org.onebusaway.api.actions.siri.impl.SiriSupportV2.Filters;
+import org.onebusaway.api.actions.siri.model.DetailLevel;
+import org.onebusaway.geospatial.model.CoordinateBounds;
+import org.onebusaway.gtfs.model.AgencyAndId;
+import org.onebusaway.gtfs.model.Trip;
+import org.onebusaway.gtfs.model.calendar.ServiceDate;
+import org.onebusaway.transit_data.model.*;
+import org.onebusaway.transit_data.model.RouteBean.Builder;
+import org.onebusaway.transit_data.model.schedule.StopTimeBean;
+import org.onebusaway.transit_data.model.trips.TripBean;
+import org.onebusaway.transit_data.services.TransitDataService;
+import org.onebusaway.transit_data_federation.impl.ExtendedCalendarServiceImpl;
+import org.onebusaway.transit_data_federation.impl.blocks.BlockIndexFactoryServiceImpl;
+import org.onebusaway.transit_data_federation.impl.blocks.BlockIndexServiceImpl;
+import org.onebusaway.transit_data_federation.impl.narrative.NarrativeServiceImpl;
+import org.onebusaway.transit_data_federation.impl.transit_graph.*;
+import org.onebusaway.transit_data_federation.model.narrative.AgencyNarrative;
+import org.onebusaway.transit_data_federation.model.narrative.TripNarrative;
+import org.onebusaway.transit_data_federation.services.beans.RouteScheduleBeanService;
+import org.onebusaway.transit_data_federation.services.blocks.BlockTripIndex;
+import org.onebusaway.transit_data_federation.services.transit_graph.*;
+import org.onebusaway.transit_data_federation.siri.SiriXmlSerializerV2;
+import org.onebusaway.util.services.configuration.ConfigurationService;
+
+///THIS TEST IS INCOMPLETE
```

Comment:
Unless you are planning to complete this test this week, i would now delete this test.  It's in source control if we come back to it.